### PR TITLE
Improved api docs

### PR DESCRIPTION
**Pull request by Ruben1701**
# Description
Split up the different endpoints using tags and added some information and context to each existing endpoint.

## Checklist before requesting a review

Please delete options that are not relevant.

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged

## Screenshots (if appropriate):
Screenshot 2023-06-15 at 12 05 25
Screenshot 2023-06-15 at 12 05 55

### CODE REVIEW

File: backend/routes/chat_routes.py [hunk 1]
```diff
@@ -30,72 +30,63 @@ def fetch_user_stats(commons, user, date):
...
-@chat_router.delete("/chat/{chat_id}", dependencies=[Depends(AuthBearer())])
+@chat_router.delete("/chat/{chat_id}", dependencies=[Depends(AuthBearer())], tags=["Chat"])
 async def delete_chat(commons: CommonsDep, chat_id: UUID):
+    """
+    Delete a specific chat by chat ID.
+    """
     delete_chat_from_db(commons, chat_id)
     return {"message": f"{chat_id}  has been deleted."}
 
-# helper method for update and create chat
```
Comment (mattzcarey) [inline backend/routes/chat_routes.py:55]: Hey, Thanks for your contribution. 

I am confused why you seem to have removed a lot of the logic for these routes?

**Review by StanGirard — DISMISSED** [review]

**Review by mattzcarey — CHANGES_REQUESTED** [review]
Add back in removed logic.

File: backend/routes/chat_routes.py (hunk 1 above)
Comment (Ruben1701) [reply backend/routes/chat_routes.py:55]: seems to have gone wrong ill fix it thanks for the review!

**Review by mattzcarey — APPROVED** [review]

**Comment by mattzcarey** [comment]
Nicely done :)

**Review by StanGirard — APPROVED** [review]

**Pull request merged**